src/mo_reader.cpp

```diff
@@ -82,11 +82,11 @@ std::string mo_file_reader::gettext(const std::string &original_string) {
 }
 
```

Comment:
// This function reads 4-byte integer, in little-endian encoding. (this is encoding we use in Yedino/Galaxy42 project)